packages/patternfly-4/react-core/src/components/Accordion/AccordionContent.tsx

```diff
@@ -16,6 +16,8 @@ export interface AccordionContentProps extends React.HTMLProps<HTMLDivElement> {
   isFixed?: boolean;
   /** Adds accessible text to the Accordion content */
   'aria-label'?: string;
+  /** Container to override the default for content */
```

Comment:
Maybe say `Component to use as content container`?

Reply:
Sure, done!